lib/Doctrine/DBAL/Schema/ForeignKeyConstraint.php

```diff
@@ -56,11 +56,11 @@ class ForeignKeyConstraint extends AbstractAsset implements Constraint
     /**
      * Initializes the foreign key constraint.
      *
-     * @param string[]     $localColumnNames   Names of the referencing table columns.
-     * @param Table|string $foreignTableName   Referenced table.
-     * @param string[]     $foreignColumnNames Names of the referenced table columns.
-     * @param string|null  $name               Name of the foreign key constraint.
-     * @param mixed[]      $options            Options associated with the foreign key constraint.
+     * @param string[]           $localColumnNames   Names of the referencing table columns.
+     * @param Table|string       $foreignTableName   Referenced table.
+     * @param array<string|null> $foreignColumnNames Names of the referenced table columns.
```

Comment:
This is the one I'm really not sure about. I wrote it because of this test:
https://github.com/doctrine/dbal/blob/155d028be084ef69110d09938c57c351a2126e5d/tests/Doctrine/Tests/DBAL/Functional/Schema/SqliteSchemaManagerTest.php#L86-L104

But it causes other issues down the line… I think there might be a bug here.

Reply:
In https://github.com/doctrine/dbal/pull/2854, `null` was replaced with an empty string.